server/athenian/api/openapi/openapi.yaml

```diff
@@ -2146,6 +2182,18 @@ components:
           type: integer
         in:
           $ref: '#/components/schemas/RepositorySet'
+        as:
+          items:
+            enum:
```

Comment:
We should reference `PullRequestWith` here instead of defining from scratch.

Reply:
Unfortunately, I wanted to do that, but I wasn't able to. Do you have any idea on who it could be done with json schema? I tried both looking at how to define an `object` whose `properties` are actually validated using an `enum` and also whether it is possible to, on the contrary, define an `enum` from the `properties` of an `object`, but I didn't find anything.

Reply:
Yep you are right this is not possible, I overfitted to the other requests :)